templates/gRPC-C++.podspec.template

```diff
@@ -60,42 +60,43 @@
         files.update(lib.get(field, []))
     return list(sorted(files))
 
-  def list_filegroup_files(expect_filegroups, groups):
-    out = []
-    for filegroup_name in expect_filegroups:
-      filegroup = filegroup_maps[filegroup_name]
-      for group in groups:
-        out += filegroup.get(group, [])
-    return list(sorted(set(out)))
-
   def filter_grpcpp(files):
     return sorted([file for file in files if not file.startswith("include/grpc++")])
 
-  grpc_private_files = list_lib_files("grpc", ("headers", "src"))
-  grpc_public_headers = list_lib_files("grpc", ("public_headers",))
-  grpc_private_headers = list_lib_files("grpc", ("headers",))
+  # TODO(jtattermusch): get rid of this hack
```

Comment:
can you document what the hack is about?

Reply:
good catch, I was going to get rid of this hack before merging.

Reply:
Removed the hack.